caasp-389-ds-image/root/usr/local/bin/entrypoint.sh

```diff
@@ -27,38 +27,69 @@ if [ ! -e "/data/config/setup.inf" ]; then
 
     echo -e "\n>> Creating localhost configuration...\n"
     cat <<EOT >> /data/config/setup.inf
-[General]
-FullMachineName=localhost.${DOMAIN_SUFFIX}
-;SuiteSpotUserID=dirsrv
-;SuiteSpotGroup=dirsrv
-;Should not run as root
-SuiteSpotUserID=root
-;Should not run as root
-SuiteSpotGroup=root
-StrictHostCheck=false
+[general]
+config_version = 2
+;defaults = 999999999
+;full_machine_name = localhost.${DOMAIN_SUFFIX}
+selinux = False
```

Comment:
selinux, start_sever and with_systemd all set as `False` is for running container? (double confirm)

Reply:
Yes, they are set to false. We started 289-ds from command instead of systemd